src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -199,6 +200,22 @@ public PersistentDataStore() {
 
     return activities;
   }
+  public List<Setting> loadSetting()throws PersistentDataStoreException{
```

Comment:
We want this to be extendable to other settings too right? Currently we are only querying for "bacground-color" it would make sense if we query for like "settings" and return all our settings.

Reply:
got it.